eos-update-server/docs/eos-update-server.conf.5

```diff
@@ -41,6 +49,21 @@ Whether to advertise and distribute updates over the local network. This may be
 \fItrue\fP or \fIfalse\fP. If \fItrue\fP, \fBeos\-update\-server\fP(8) and
 \fBeos\-updater\-avahi\fP(8) are enabled; otherwise, they will both refuse to
 advertise or distribute updates.
+\"
+.SH [Repository 0–9] SECTION OPTIONS
```

Comment:
Also, if we decide to stay with the numbering then do you have any ideas how to make the `[Repository 0-9]` format clearer? Currently my first thought was "only max 10 repos are supported?".